__tests__/example.query.response.fail.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Nit: This file seems like a valid response to me?

Reply:
The request ID should be a string here